src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Represents a user interface that the user can input commands to and read from.
+ */
+class Ui {
+    private static final Scanner sc = new Scanner(System.in);
+
+    /**
+     * Returns next line of input from user.
+     *
+     * @return Next line.
+     */
+    static String nextLine() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Shows output to user.
+     *
+     * @param lines lines to show user.

```

Comment:
Consider using capital letters at the start of the sentence here for consistency with the rest of the Javadocs?

```suggestion
     * @param lines Lines to show user.
```

Reply:
Nice catch! You have a sharp eye :)